Dockerfile

```diff
@@ -3,15 +3,6 @@ FROM python:3.6.8-stretch
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-ENV PATH /usr/local/nvidia/bin/:$PATH
```

Comment:
So none of these lines are required to make GPUs on Docker work?

Reply:
They are--but they are in the `nvidia/cuda:10.0-base-ubuntu18.04` base image.